client/blocks/color-scheme-picker/constants.js

```diff
@@ -26,7 +26,15 @@ import sunsetImg from 'calypso/assets/images/color-schemes/color-scheme-thumbnai
 export default function ( translate ) {
 	return [
 		{
-			label: translate( 'Default' ),
+			label: translate( 'Modern' ),
+			value: 'modern',
+			thumbnail: {
+				cssClass: 'is-modern',
+				imageUrl: modernImg,
+			},
+		},
+		{
+			label: translate( 'Classic Dark' ),
```

Comment:
Should Classic Dark be moved down to after Classic Bright in order to remain in alphabetical order or is it intentional to prioritise it?

Reply:
Good catch :) Didn't notice that they are sorted in alphabetical order.

Reply:
Oh yes! Nice catch, thank you!